packages/worker/src/replay.ts

```diff
@@ -60,7 +53,11 @@ export type ReplayHistoriesIterable = AsyncIterable<HistoryAndWorkflowId> | Iter
  *
  * @internal
  */
-export function handleReplayEviction(evictJob: RemoveFromCache, workflowId: string, runId: string): ReplayError | null {
+export function evictionReasonToReplayError(
+  evictJob: RemoveFromCache,
+  workflowId: string,
```

Comment:
Do you see any real reason to keep `workflowId` and `runId` in `ReplayError`? They are already in `ReplayResult`...

Also, in `ReplayError` (can't put a comment there), `workflowId` and `isNonDeterminism` are both `readonly`, but `runId` isn't. They should be coherent.

Reply:
I'll fix that, thanks for noticing.

Reply:
`ReplayError` is used for replaying a single history but I think it's fine that it doesn't give back the workflow ID and run ID.